ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/DeferralForm.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import {
+    Bullseye,
+    Button,
+    DatePicker,
+    Flex,
+    FormGroup,
+    Form,
```

Comment:
Small observation: 2 pairs seem to have reversed order.

Although a lint rule would be better than review comments.

Reply:
Agreed, possibly https://github.com/trivago/prettier-plugin-sort-imports?